packages/express/test/tck.ts

```diff
@@ -0,0 +1,107 @@
+import http from 'node:http';
+import { AddressInfo } from 'node:net';
+import path from 'node:path';
+
+import { after } from 'mocha';
+import express from 'express';
+import { GenericContainer, TestContainers } from 'testcontainers';
+import { assert } from 'chai';
+
+import httpwasm from '../index.js';
+
+const tckImage = 'ghcr.io/http-wasm/http-wasm-tck:sha-e25f1a2';
+
+describe('tck', async function () {
+  this.timeout(10000);
+
+  let server: http.Server;
+  let port: number;
+
+  before(async function () {
+    const wasmPath = path.resolve('..', '..', 'build', 'tck.wasm');
+
+    const tck = await new GenericContainer(tckImage)
+      .withCommand(['extract-guest', '/out/tck.wasm'])
+      .withBindMounts([
+        {
+          source: path.resolve('..', '..', 'build'),
+          target: '/out',
+          mode: 'rw',
+        },
+      ])
+      .start();
+
+    const app = express();
+    const mw = await httpwasm({ wasmPath });
+    app.use(mw);
+
+    app.all('/*', (req, res) => {
+      res.setHeader('x-httpwasm-next-method', req.method);
+      res.setHeader('x-httpwasm-next-uri', req.url);
+      req.headers;
+      for (const [key, value] of Object.entries(req.headers)) {
+        if (!value) {
+          continue;
+        }
+        const values = Array.isArray(value) ? value : [value];
+        for (let i = 0; i < values.length; i++) {
+          res.setHeader(`x-httpwasm-next-header-${key}-${i}`, values[i]);
+        }
+      }
+      res.sendStatus(200);
```

Comment:
in general for tests I prefer to use a code different than 200. Some frameworks (maybe express too) can chose to return 200 even if status code hasn't being sent so better to use 202 o 201 to do further assertions.

Reply:
This would need changing the TCK so will leave it for now and think on whether it's useful